libexec/rtld-elf/rtld_c18n.h

```diff
@@ -118,6 +118,44 @@ struct stk_table {
 
 #define	COMPART_ID_MAX		index_to_cid((stk_table_index) { -1 })
 
+#include "rtld_c18n_machdep.h"
+
+struct trusted_frame {
+	struct compart_state state;
+	/*
+	 * INVARIANT: This field contains the top of the caller's stack when the
+	 * caller was last entered.
+	 */
+	void *osp;
+	/*
+	 * Address of the previous trusted frame
+	 */
+	struct trusted_frame *previous;
+	/*
+	 * Compartment ID of the caller
+	 */
+	stk_table_index caller;
+	/*
+	 * Zeros
+	 */
+	uint16_t zeros;
```

Comment:
Not specifically related to this diff, but what is "zeros" and why is it here?

Reply:
This is just padding space that must be filled with zeros.

Reply:
Why?

Reply:
@jrtc27 So that when you do a wide load (e.g. a capability load) from the `caller` field, you can extract the caller for free by using a narrower register (i.e. the `w` register on Morello).

Reply:
I will document it.